assets/js/modules/search-console/dashboard/dashboard-widget.js

```diff
@@ -236,4 +243,4 @@ class GoogleSitekitSearchConsoleDashboardWidget extends Component {
 	}
 }
 
-export default GoogleSitekitSearchConsoleDashboardWidget;
+export default withSelect( ( select ) => ( { dateRange: select( STORE_NAME ).getDateRange() } ) )( GoogleSitekitSearchConsoleDashboardWidget );
```

Comment:
Same as previous comment; it's a bit long as a single line 🙂

Reply:
Updated.